pinot-plugins/pinot-file-system/pinot-gcs/src/main/java/org/apache/pinot/plugin/filesystem/GcsPinotFS.java

```diff
@@ -325,6 +326,7 @@ public long length(URI fileUri) throws IOException {
   @Override
   public void copyToLocalFile(URI srcUri, File dstFile) throws Exception {
     LOGGER.info("Copy {} to local {}", srcUri, dstFile.getAbsolutePath());
+    FileUtils.forceMkdir(dstFile.getParentFile());
```

Comment:
should we check the existence before creating? will forceMkdir throw an exception if the directory exists?

Reply:
No. This util is idempotent. Doesn't throw any exception even if directory exists.